app/src/main/java/edu/cascadia/mobas/pollinatorpathway/ui/login/UserDatabase.java

```diff
@@ -0,0 +1,27 @@
+package edu.cascadia.mobas.pollinatorpathway.ui.login;
+
+import android.content.Context;
+
+import androidx.room.Database;
+import androidx.room.Room;
+import androidx.room.RoomDatabase;
+
+@Database(entities = {Profile.class}, version = 1)
+public abstract class UserDatabase extends RoomDatabase {
```

Comment:
I wonder if you want to make the database class name a bit more general, since it's not only user data.
Perhaps PathwayDatabase or PnwppDatabase? 

Just a suggestion, it's up to you.